static/js/app.js

```diff
@@ -104,28 +149,40 @@ async function deleteEvent(eventId) {
   }
 }
 
+/**
+ * Updates an existing calendar event.
+ * @async
+ * @param {string} eventId - The unique identifier of the event to update.
+ * @param {Object} updatedData - The updated event data.
+ */
 async function updateEvent(eventId, updatedData) {
   try {
+    const timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+    updatedData.timeZone = timeZone;
     await axios.put(`/api/calendar/events/${eventId}`, updatedData);
     await renderCalendar(currentYear, currentMonth);
   } catch (error) {
     console.error('Error updating event:', error);
   }
 }
 
+/**
+ * Sets up the event update form with the specified event's data.
+ * @param {Object} event - The event object.
+ */
 function setupEventUpdateForm(event) {
   document.getElementById('event-title').value = event.summary || '';
   document.getElementById('event-date').value = event.start.dateTime.split('T')[0];
   document.getElementById('event-start-time').value = event.start.dateTime.split('T')[1].slice(0, 5);
   document.getElementById('event-end-time').value = event.end.dateTime.split('T')[1].slice(0, 5);
   document.getElementById('event-location').value = event.location || '';
   document.getElementById('event-description').value = event.description || '';
-
+  const timeZone = event.start.timeZone || 'UTC';
```

Comment:
Do we need to assign this as a variable or can we just run event.start.timeZone? Just wondering since it's throwing a codacy error, but it could be a mistake on codacy's end

Reply:
Yes the line should have been removed. I forgot. Thank you for pointing it out